homeassistant/components/calendar/todoist.py

```diff
@@ -83,7 +87,9 @@
     vol.Optional(PROJECT_NAME, default='inbox'): vol.All(cv.string, vol.Lower),
     vol.Optional(LABELS): cv.ensure_list_csv,
     vol.Optional(PRIORITY): vol.All(vol.Coerce(int), vol.Range(min=1, max=4)),
-    vol.Optional(DUE_DATE): cv.string,
+    vol.Optional(DATE_STRING): cv.string,
+    vol.Optional(DATE_LANG): vol.All(cv.string, vol.Length(min=2, max=2)),
+    vol.Optional(DUE_DATE): cv.string
```

Comment:
Since specifying both a date string and a due date is not possible (see below), this should probably already be caught in validation?